content/build-tool.md

```diff
@@ -79,7 +79,15 @@ One difference between pre-published packages and local app packages is that the
 
 <h4 id="npm-adding">Adding packages to your app</h4>
 
-Meteor 1.3 will have seamless integration with NPM, and you will be able to simply `npm install` these packages into your app directory. Until then, the easiest way to use NPM packages in your app is [`meteorhacks:npm`](https://atmospherejs.com/meteorhacks/npm).
+To install an NPM package into your app, you can simply run NPM's standard install command:
```

Comment:
> NPM

> It's pretty backwards compatible, right?

Top kek.

No but seriously libraries such as Relay don't work properly with NPM 3 sometimes. This is a fact: https://github.com/facebook/relay/issues/832

Doesn't matter though - I think we should probably recommend NPM 3, and maybe people can figure it out.